app/controllers/orders_controller.rb

```diff
@@ -18,15 +18,20 @@ def create
           price: item.price
           })
       end
+      order.item_orders.each do |item_order|
+        item = Item.find(item_order.item_id)
+        item.reduce_quantity(item_order.quantity)
+        item.save
+      end
```

Comment:
There's probably a better way of reducing item quantity. We can ask Meg at check ins.